specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/stable/2022-05-13/confidentialledger.json

```diff
@@ -828,10 +840,16 @@
     },
     "CollectionArray": {
       "description": "A list of identifiers for ledger collections.",
-      "items": {
-        "$ref": "#/definitions/Collection"
-      },
-      "type": "array"
+      "properties": {
```

Comment:
I think there's a problem here -- you are changing the return type from an array (I think we discussed this before) to an object type. An object type is definitely preferable, but it will require a change in your service.  Are you able to make that change?